CoreDependencies/nbproject/project.xml

```diff
@@ -2563,136 +2559,136 @@
                 <binary-origin>release/modules/ext/jsr305-3.0.2.jar</binary-origin>
             </class-path-extension>
             <class-path-extension>
-                <runtime-relative-path>ext/jt-affine-1.1.14.jar</runtime-relative-path>
```

Comment:
@formalhaut69 are you branching off the latest master? In theory this file shouldn't change and the dependencies are going from 1.1.14 to 1.1.12. I merged in #632 yesterday and I thought I was on the latest master when I did this. Curios to know what's gone wrong here.

Reply:
Yeah I actually made my changes on master and then made a new branch for the pull request. I also checked to see if my branch is up to date and it is. Do you have any suggestions for how to manually fix this @arcturus2 ?

Reply:
Can you try to open the `ProjectUpdater` code, do a clean build of it and then run `update-dependencies-clean-build` ?

The ProjectUpdater code doesn't get rebuild if its already compiled and perhaps my changes to ProjectUpdater to sort the dependencies has not taken effect so this would help.